src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -74,7 +114,20 @@ public LocalDate addWeeks(LocalDate localDate, Integer numberOfWeeks) {
      * - "someDate is today" - если someDate - сегодня
      */
     public String beforeOrAfter(LocalDate someDate) {
-        return someDate + "is today";
+        LocalDate curDate = LocalDate.now();
+        if (someDate == null) {
+            return null;
+        }
+        if (someDate.isAfter(curDate)) {
+            return someDate + " is after " + curDate;
+        }
+        if (someDate.isBefore(curDate)) {
+            return someDate + " is before " + curDate;
+        }
+        if (someDate.isEqual(curDate)) {
```

Comment:
if (someDate.isEqual(curDate)) {
            return someDate + " is today";
        }
        return "";
-->
return someDate + " is today";

Reply:
OK